public/buttons.html

```diff
@@ -114,6 +114,19 @@ <h1>PayPal Buttons Example</h1>
                 errorObject: error,
               });
             },
+            onShippingChange(data) {
+              console.log(data);
```

Comment:
Do we want to keep this console log here?  Not sure if it's just left over from debugging.